Gemfile

```diff
@@ -60,7 +60,7 @@ end
 
 
 group :development, :test, :testing do
-  gem 'factory_girl_rails', '~> 4.9.0'
+  gem "factory_bot_rails"
```

Comment:
It looks like you need to commit your `Gemfile.lock` according to the Travis CI failure.